### ISSUE TITLE
Release with mercurius 14 as a supported peer dependency?

### ISSUE THREAD

**Issue by twalling** [mercurius-js/mercurius-typescript#846]
Would it be possible to get another release since mercurius 14 was released and I see that it was already added as a peer dependency on the main branch?

**Comment by Drilmo** [comment]
+1

**Comment by Drilmo** [comment]
If you want you can try this workaround in your package.json until we get the new version : 
```json
"overrides": {
    "mercurius-codegen": {
      "mercurius": "$mercurius"
    }
  }
```

**Comment by luthfi-hh** [comment]
Any update on this?

**Comment by PabloSzx** [comment]
Published v5.0.5 with mercurius v14 in peer dependency, sorry for the very late update
